Makefile

```diff
@@ -1,11 +1,14 @@
-.PHONY: pytest nox wc clean clobber upload
+.PHONY: pytest ruff nox wc clean clobber upload
 
 XARGS := xargs $(shell test $$(uname) = Linux && echo -r)
 VERSION := $(shell grep __version__ src/dark/__init__.py | cut -f2 -d'"')
 
 pytest:
 	uv run pytest
 
+ruff:
+	ruff check --fix --extend-select I
```

Comment:
You are using `uv run` for everything else, why not for `ruff`?

Reply:
Fixed, thanks.